src/conditions/base/time.ts

```diff
@@ -1,24 +1,18 @@
 import Joi from 'joi';
 
-import { SUPPORTED_CHAINS } from '../const';
+import { RpcCondition, rpcConditionSchema } from './rpc';
 
-import { Condition } from './condition';
-import { returnValueTestSchema } from './schema';
+const BLOCKTIME_METHOD = 'blocktime';
 
-export class TimeCondition extends Condition {
-  // TODO: This is the only condition that uses defaults, and also the only condition that uses `method` in order
-  //   to determine the schema. I.e. the only method that used `METHOD = 'blocktime'` in `nucypher/nucypher`.
-  // TODO: Consider introducing a different field for this, e.g. `conditionType` or `type`. Use this field in a
-  //  condition factory.
+const timeConditionSchema = {
+  ...rpcConditionSchema,
```

Comment:
There is a separation between the subclassing the rpc.schema and subclassing the rpc condition object. 

The issue with subcalssing the rpc.schema is that `parameters` are allowed, and for time condition parameters should not be allowed. 

In python, TimeCondition subclasses RPCCondition but the schemas do not subclass each other - what's the best way of doing that in ts?